contrib/dockerfile_build.bzl

```diff
@@ -89,7 +89,7 @@ exports_files(["{}"])
         if build_result.return_code:
             fail("Image build command failed: {} ({})".format(
                 build_result.stderr,
-                " ".join(command),
+                " ".join([str(e) for e in command]),
```

Comment:
why is this needed, shouldn't all elements in command already be strings? If not, its probably better to make them strings as you are putting them in the command instead of converting them here

Reply:
Done.
Some of the elements were of type `path`.